.github/ISSUE_TEMPLATE/BUG_REPORT.md

```diff
@@ -0,0 +1,35 @@
+---
+name: '🐞 Bug Report'
+about: Create a report to help us improve
+title: ''
+labels: ['type:Bug', 'status:Unconfirmed']
+assignees: ''
+---
+
+<!--
+Found a bug? Please fill out the sections below. 👍
+-->
+
+### Issue Summary
+
+<!--
+A summary of the issue.
+-->
+
+### Steps to Reproduce
+
+1. (for example) Start a new project with `python manage.py runserver`
+2. Edit models.py as follows...
+3. ...
+
+Any other relevant information. For example, why do you consider this a bug and what did you expect to happen instead?
+
+-   I have confirmed that this issue can be reproduced as described on a fresh gymkhana project: (yes / no)
+
+### Technical details
+
+-   Python version: Run `python --version`.
+-   Django version: Look in your requirements.txt, or run `pip show django | grep Version`.
+-   Node version: Run `node -v`
+-   React version: Run `npm view react version`
+-   Browser version: You can use https://www.whatsmybrowser.org/ to find this out.
```

Comment:
Again, provide users with the space to put their answers.

```suggestion
| Name | Executable | Version |
| --- | ---| --- |
| Python version | `python --version` | VERSION |
| Django version | `pip show django | grep Version` | VERSION |
| Node version | `node -v` | VERSION |
...so on
```

Reply:
This seems reasonable for me